test/e2e/regression/api_test.go

```diff
@@ -184,7 +186,7 @@ func (s *APIRegressionTestSuite) TestProjectAPI() {
 		_, err := s.testBench.Client.UpdateProject(ctxOrgAdminAuth, &shieldv1beta1.UpdateProjectRequest{
 			Id: "random",
 			Body: &shieldv1beta1.ProjectRequestBody{
-				Name:  "new project",
+				Name:  "new-project",
```

Comment:
Name can contain spaces.

Reply:
Should I update the proto for the same. project, group and organisation names too have the same regex. "^[A-Za-z0-9_-]+$" https://github.com/odpf/proton/blob/main/odpf/shield/v1beta1/shield.proto#L943